tests/py/dynamo/conversion/test_embedding_bag_aten.py

```diff
@@ -408,6 +410,86 @@ def forward(self, weight, indices, offsets):
             propagate_shapes=True,
         )
 
+    @parameterized.expand(
+        [
+            param(
+                # 1d_indices_mode_0_with_per_sample_weights
+                weights=torch.randn((5, 2), dtype=torch.float32),
+                dynamic_shapes={
+                    "weights": {0: torch.export.Dim("dyn_dim", min=2, max=6)},
+                    "indices": {},
+                    "offsets": {},
+                },
+                # weights_max_dim=6,
+                indices=torch.tensor([1, 2, 4], dtype=torch.int32),
+                offsets=torch.tensor([0, 2, 3], dtype=torch.int32),
+                mode=0,
+                per_sample_weights=torch.randn((3,), dtype=torch.float32),
+            ),
+            param(
+                # 1d_indices_mode_1_without_per_sample_weights
+                weights=torch.randn((5, 2), dtype=torch.float32),
+                dynamic_shapes={
+                    "weights": {
+                        0: torch.export.Dim("dyn_dim", min=2, max=8),
+                        1: torch.export.Dim("dyn_dim_1", min=1, max=3),
+                    },
+                    "indices": {},
+                    "offsets": {},
+                },
+                indices=torch.tensor([1, 2, 4, 2, 3, 4], dtype=torch.int32),
+                offsets=torch.tensor([0, 2, 4], dtype=torch.int32),
+                mode=1,
+                per_sample_weights=None,
+            ),
+        ]
+    )
+    def test_embedding_bag_with_weights_dynamic_shape(
+        self, weights, dynamic_shapes, indices, offsets, mode, per_sample_weights
+    ):
+        class EmbeddingBag(torch.nn.Module):
+            def forward(self, weights, indices, offsets, per_sample_weights=None):
+                return torch.ops.aten._embedding_bag.default(
+                    weight=weights,
+                    indices=indices,
+                    offsets=offsets,
+                    per_sample_weights=per_sample_weights,
+                    scale_grad_by_freq=False,
+                    mode=mode,
+                    sparse=False,
+                    include_last_offset=False,
+                    padding_idx=-1,
+                )
+
+        if per_sample_weights is None:
+            inputs = (weights, indices, offsets)
+        else:
+            inputs = (weights, indices, offsets, per_sample_weights)
+        mod = EmbeddingBag()
+
+        if per_sample_weights is not None:
+            dynamic_shapes["per_sample_weights"] = {}
+        fx_mod = torch.export.export(mod, inputs, dynamic_shapes=dynamic_shapes)
+        trt_mod = torch_tensorrt.dynamo.compile(
+            fx_mod, inputs=inputs, enable_precisions=torch.float32, min_block_size=1
+        )
+
+        with torch.no_grad():
+            cuda_inputs = []
+            for i in inputs:
```

Comment:
Instead of using the same inputs that were used to compile, can you try with another shape within the dynamic shape range ? Like in the first example, it would be (6, 2)